Gemfile.lock

```diff
@@ -201,6 +205,7 @@ DEPENDENCIES
   binding_of_caller
   byebug
   coffee-rails (~> 4.2)
+  dotenv-rails
```

Comment:
We don't want this in two places, so we'll want to remove it before the next commit.